src/components/invest/investslug/components/InvestSection.tsx

```diff
@@ -0,0 +1,73 @@
+import Link from 'next/link'
+import React from 'react'
+import { ProjectInvestData } from '../data/ProjectInvestData'
+
+function formatNumberWithCommas(num: number): string {
+  return num.toLocaleString()
+}
+
+type Props = {
+  project: ProjectInvestData
+}
+
+// TODO: Ask how to refactor this component on review, to put it into separete files, maybe just one big one with sections?
+
+export const InvestSection: React.FC<Props> = ({ project }) => {
+  return (
+    <div className='flex flex-col p-5 gap-6 sticky top-[100px] w-[28%] h-[550px] min-w-[280px] fixed top-100 overflow-x-auto overflow-y-auto bg-[#1f222a] justify-start items-between rounded-xl'>
```

Comment:
one note about bg color. Before putting it like this, please check if its already existing in our tailwind config.
For example this one here is `bg-grey-500`
You have sticky and fixed, I think you would like to apply `sticky` only?
Do we really need these classes: `overflow-x-auto overflow-y-auto`

Again if possible, I would like if its possible to avoid setting fixed height, only use `max-height` and then use `flex` and proper `gap` or `justify-between` with gaps (in this case I can see on figma) to make it looks like on figma as much as possible